examples/stable_diffusion_xl/vanilla_finetune.md

```diff
@@ -0,0 +1,152 @@
+# Vanilla finetune
+
+**⚠️ Note**: We currently only support fine-tuning on Ascend 910*
+
+## Preparation
+
+### Dependency
+
+Make sure the following frameworks are installed.
+
+- mindspore 2.2.10 (Ascend 910*)
+- openmpi 4.0.3 (for distributed mode)
+
+```shell
+pip install -r requirements.txt
+```
+
+### Pretrained models
+
+Download the official pre-train weights from huggingface, convert the weights from `.safetensors` format to Mindspore `.ckpt` format, and put them to `./checkpoints/` folder. Please refer to SDXL [weight_convertion.md](./weight_convertion.md) for detailed steps.
+
+### Dataset Preparation for Fine-Tuning
+
+The text-image dataset for fine-tuning should follow the file structure below
+
+<details onclose>
+
+```text
+dir
+├── img1.jpg
+├── img2.jpg
+├── img3.jpg
+└── img_txt.csv
+```
+
+img_txt.csv is the annotation file in the following format
+```text
+dir,text
+img1.jpg,a cartoon character with a potted plant on his head
+img2.jpg,a drawing of a green pokemon with red eyes
+img3.jpg,a red and white ball with an angry look on its face
+```
+
+For convenience, we have prepared two public text-image datasets obeying the above format.
+
+- [pokemon-blip-caption dataset](https://openi.pcl.ac.cn/jasonhuang/mindone/datasets), containing 833 pokemon-style images with BLIP-generated captions.
+- [Chinese-art blip caption dataset](https://openi.pcl.ac.cn/jasonhuang/mindone/datasets), containing 100 Chinese art-style images with BLIP-generated captions.
+
+To use them, please download `pokemon_blip.zip` or `chinese_art_blip.zip` from the [openi dataset website](https://openi.pcl.ac.cn/jasonhuang/mindone/datasets). Then unzip.
+
+</details>
+
+
+## Finetuning
```

Comment:
The content below is dreambooth finetune, not vanilla finetune.

Reply:
Okay, I'll delete it first and add it later